src/uwtools/tests/fixtures/hello_workflow_tags.yaml

```diff
@@ -0,0 +1,11 @@
+  workflow:
+    attrs:
+      realtime: false
+      scheduler: slurm
```

Comment:
Probably add 'cyclethrottle' here too just to make sure all the tags are going in and being checked?